src/PolyhedralGeometry/helpers.jl

```diff
@@ -478,37 +481,56 @@ _detect_default_field(::Type{QQFieldElem}, p::Polymake.BigObject) = QQ
 _detect_default_field(::Type{Float64}, p::Polymake.BigObject) = AbstractAlgebra.Floats{Float64}()
 
 function _detect_default_field(::Type{T}, p::Polymake.BigObject) where T<:FieldElem
-    # we only want to check existing properties
-    f = x -> Polymake.exists(p, string(x))
-    propnames = intersect(propertynames(p), [:INPUT_RAYS, :POINTS, :RAYS, :VERTICES, :VECTORS, :INPUT_LINEALITY, :LINEALITY_SPACE, :FACETS, :INEQUALITIES, :EQUATIONS, :LINEAR_SPAN, :AFFINE_HULL])
-    i = findfirst(f, propnames)
-    # find first OscarNumber wrapping a FieldElem
-    while !isnothing(i)
-        prop = getproperty(p, propnames[i])
-        for el in prop
-            on = Polymake.unwrap(el)
-            if on isa T
-                return parent(on)
-            end
-        end
-        i = findnext(f, propnames, i + 1)
+  # we only want to check existing properties
+  propnames = intersect(Polymake.list_properties(p), ["INPUT_RAYS", "POINTS", "RAYS", "VERTICES", "VECTORS", "INPUT_LINEALITY", "LINEALITY_SPACE", "FACETS", "INEQUALITIES", "EQUATIONS", "LINEAR_SPAN", "AFFINE_HULL"])
```

Comment:
```suggestion
  propnames = String.(intersect(Polymake.list_properties(p), ["INPUT_RAYS", "POINTS", "RAYS", "VERTICES", "VECTORS", "INPUT_LINEALITY", "LINEALITY_SPACE", "FACETS", "INEQUALITIES", "EQUATIONS", "LINEAR_SPAN", "AFFINE_HULL"]))
```
For some reason intersect returns C++ strings on julia 1.6 which is not accepted by getproperty.